x-pack/plugins/reporting/public/components/report_listing.tsx

```diff
@@ -0,0 +1,488 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage, InjectedIntl, injectI18n } from '@kbn/i18n/react';
+import { get } from 'lodash';
+import moment from 'moment';
+import React, { Component } from 'react';
+import { Subscription } from 'rxjs';
+
+import {
+  EuiBasicTable,
+  EuiButtonIcon,
+  EuiPageContent,
+  EuiSpacer,
+  EuiText,
+  EuiTextColor,
+  EuiTitle,
+  EuiToolTip,
+} from '@elastic/eui';
+
+import { ToastsSetup, ApplicationStart } from '../../';
+import { LicensingPluginSetup, LICENSE_CHECK_STATE, ILicense } from '../../../licensing/public';
+import { Poller } from '../../common/poller';
+import { JobStatuses, JOB_COMPLETION_NOTIFICATIONS_POLLER_CONFIG } from '../../constants';
+import { JobQueueClient, JobQueueEntry } from '../lib/job_queue_client';
+import { ReportErrorButton } from './report_error_button';
+import { ReportInfoButton } from './report_info_button';
+
+interface Job {
+  id: string;
+  type: string;
+  object_type: string;
+  object_title: string;
+  created_by?: string;
+  created_at: string;
+  started_at?: string;
+  completed_at?: string;
+  status: string;
+  statusLabel: string;
+  max_size_reached: boolean;
+  attempts: number;
+  max_attempts: number;
+  csv_contains_formulas: boolean;
+}
+
```

Comment:
The signature for this component has changed pretty drastically, as we now pass down license-observables, and other clients, through props.